.pre-commit-config.yaml

```diff
@@ -12,6 +12,10 @@ repos:
     -   id: check-toml
     -   id: check-xml
     -   id: check-yaml
+        args:
+          # https://github.com/pre-commit/pre-commit-hooks/issues/701
+          # Necessary so that non-standard tags do not throw failures.
+          - '--unsafe'
```

Comment:
Would 
```suggestion
    -   id: check-yaml
        exclude: mkdocs.yml
    -   id: check-yaml
        name: check-xml (mkdocs)
        files: mkdocs.yml
        args:
        - --unsafe
```

Reply:
Working on it, should only be a second.

Reply:
You can use the "Commit suggestion" button if this works for you.

Reply:
@scottschreckengaust Done.